packages/core/test/unit/stryker.spec.ts

```diff
@@ -51,7 +53,9 @@ describe(Stryker.name, () => {
       .withArgs(commonTokens.getLogger)
       .returns(getLoggerStub)
       .withArgs(coreTokens.temporaryDirectory)
-      .returns(temporaryDirectoryMock);
+      .returns(temporaryDirectoryMock)
+      .withArgs(commonTokens.options)
+      .returns(strOptions);
```

Comment:
```suggestion
      .returns(testInjector.options);
```

You can use the `testInjector.options` here. It is a helper object that contains options you can used. They will be cleaned after each test (reset to default)